Contributions/Dice Roll game/README.md

```diff
@@ -0,0 +1,14 @@
+# Your Project Name
+Double Dice Simulator
+
+### Description
+
+A simple project which will ask the user if he or she wants to roll the dices or not. If yes, then the computer will roll a random dices and show its output.
+- Highly useful while playing dice game with your friends.
+- Even more useful if someone doubts the physical dices provided.
+
```

Comment:
Can you add some Screenshots of your working application.

Gifs aur video also works well

Reply:
[Screencast from 14-10-22 08:21:59 PM IST.webm](https://user-images.githubusercontent.com/97968307/195877249-26933c01-dcd3-4353-a379-ef7f61b4e513.webm)

Reply:
@Abbhiishek this is a short video on the project

Reply:
Nice demonstration 😮 

- Insert the video in `README.md` file

Reply:
@Abbhiishek  Inserted the video in Readme. Please take a look.